### PR TITLE
API: Add "authorVerified" to the video endpoint

### PR DESCRIPTION
**Pull request by RadoslavL**
This contributes to #3323.

### CODE REVIEW

**Comment by SamantazFox** [comment]
Please use `make format` to run crystal's auto-formatting tool!

**Comment by RadoslavL** [comment]
> Please use `make format` to run crystal's auto-formatting tool!

Thank you for the tip!

File: src/invidious/jsonify/api_v1/video_json.cr [hunk 1]
```diff
@@ -39,6 +39,7 @@ module Invidious::JSONify::APIv1
       json.field "author", video.author
       json.field "authorId", video.ucid
       json.field "authorUrl", "/channel/#{video.ucid}"
+      json.field "authorVerified", video.info["authorVerified"]
```
Comment (SamantazFox) [inline src/invidious/jsonify/api_v1/video_json.cr:42]: There is an instance method to access that value directly ^^
```suggestion
      json.field "authorVerified", video.author_verified
```

**Review by SamantazFox — APPROVED** [review]

**Comment by SamantazFox** [comment]
Thanks for contributing to invidious ^^

**Pull request merged**